crates/rattler_upload/src/upload/prefix.rs

```diff
@@ -152,12 +155,16 @@ pub async fn upload_package_to_prefix(
             .to_string_lossy()
             .to_string();
         let file_size = package_file.metadata().into_diagnostic()?.len();
-        let url = prefix_data
+        let mut url = prefix_data
             .url
             .join(&format!("api/v1/upload/{}", prefix_data.channel))
             .into_diagnostic()?;
 
-        // Generate attestation if we're using trusted publishing and it was requested
+        if prefix_data.force.is_enabled() {
+            url.set_query(Some("force=true"));
```

Comment:
Can we use [`query_pairs_mut`](https://docs.rs/url/latest/url/struct.Url.html#method.query_pairs_mut) and insert that way?